TechnicoBackend/Controllers/RepairController.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using TechnicoBackend.Services;
+using TechnicoBackend.Models;
+using TechnicoBackend.DTOs;
+
+namespace TechnicoBackend.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class RepairController : ControllerBase
+    {
+        private readonly RepairService _repairService;
+        private readonly PropertyService _propertyService; // Προσθήκη PropertyService
```

Comment:
Generally consider using English language for comments, variables etc, since the code could be reviewed/revised by non-greek speaking individuals in the future.